src/genpy/message.py

```diff
@@ -106,10 +107,10 @@ def strify_message(val, indent='', time_offset=None, current_time=None, field_fi
     elif type_ in (int, long, float, bool):
         return str(val)
     elif isstring(val):
-        #TODO: need to escape strings correctly
         if not val:
             return "''"
-        return val
+        # escape strings for use in yaml file using yaml dump with default style to avoid trailing "...\n"
+        return yaml.dump(val, default_style='"').rstrip('\n')
```

Comment:
After reading the comment I have one question: with `default_style='"'` does it still need the `rstrip('\n')`?